### ISSUE TITLE
Build #1356 - [SEVERE] Could not pass event BlockFromToEvent to GriefPrevention v7.6.2

### ISSUE THREAD

**Issue by DrFestus** [MinimallyCorrect/TickThreading#342]
Could be MCPC related though but its something to do with already decorating

[SEVERE] Could not pass event BlockFromToEvent to GriefPrevention v7.6.2
org.bukkit.event.EventException
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:363)
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
    at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:477)
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java)
    at aky.b(BlockFlowing.java:220)
    at ace.a(WorldGenLiquids.java:81)
    at zc.a(BiomeDecorator.java:403)
    at zc.a(BiomeDecorator.java:203)
    at twilightforest.biomes.TFBiomeDecorator.a(TFBiomeDecorator.java:175)
    at yy.a(BiomeGenBase.java:366)
    at twilightforest.world.ChunkProviderTwilightForest.a(ChunkProviderTwilightForest.java:927)
    at org.bukkit.craftbukkit.v1_4_R1.generator.NormalChunkGenerator.a(NormalChunkGenerator.java:46)
    at im.a(ThreadedChunkProvider.java:615)
    at zz.a(Chunk.java:1318)
    at im.getChunkAtInternal(ThreadedChunkProvider.java:520)
    at im.getChunkAt(ThreadedChunkProvider.java:400)
    at me.nallar.patched.storage.ThreadedChunkProvider$ChunkLoadRunnable.run(ThreadedChunkProvider.java:739)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at me.nallar.tickthreading.util.FakeServerThread.run(FakeServerThread.java:16)
Caused by: java.lang.RuntimeException: Already decorating!!
    at zc.a(BiomeDecorator.java:195)
    at twilightforest.biomes.TFBiomeDecorator.a(TFBiomeDecorator.java:175)
    at yy.a(BiomeGenBase.java:366)
    at twilightforest.world.ChunkProviderTwilightForest.a(ChunkProviderTwilightForest.java:927)
    at org.bukkit.craftbukkit.v1_4_R1.generator.NormalChunkGenerator.a(NormalChunkGenerator.java:46)
    at im.a(ThreadedChunkProvider.java:615)
    at zz.a(Chunk.java:1313)
    at im.getChunkAtInternal(ThreadedChunkProvider.java:520)
    at im.getChunkAt(ThreadedChunkProvider.java:400)
    at im.c(ThreadedChunkProvider.java:350)
    at org.bukkit.craftbukkit.v1_4_R1.CraftWorld.getChunkAt(CraftWorld.java:200)
    at org.bukkit.craftbukkit.v1_4_R1.CraftWorld.getBlockAt(CraftWorld.java:164)
    at org.bukkit.craftbukkit.v1_4_R1.block.CraftBlock.getRelative(CraftBlock.java:161)
    at org.bukkit.craftbukkit.v1_4_R1.block.CraftBlock.getRelative(CraftBlock.java:169)
    at org.bukkit.craftbukkit.v1_4_R1.block.CraftBlock.getRelative(CraftBlock.java:165)
    at org.bukkit.event.block.BlockFromToEvent.getToBlock(BlockFromToEvent.java:49)
    at me.ryanhamshire.GriefPrevention.BlockEventHandler.onBlockFromTo(Unknown Source)
    at sun.reflect.GeneratedMethodAccessor150.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:361)

**Comment by joey19982** [comment]
As of build #1361 this is still happening when world unloading is enabled.

**Comment by LunNova** [comment]
Could you give the log? It's probably a different issue happening when calling the same event.

**Comment by Maximuscr31** [comment]
I am having this same issue with L19. Could we get this fixed for the legacy build as well?

**Comment by LunNova** [comment]
@Maximuscr31

Are you sure you're replying to the right issue? Your response doesn't really make sense in this context...

**Comment by Maximuscr31** [comment]
I am gettting the same error with passing events to griefprevention but I just noticed he is using 7.6.2 and I am as well which is for 1.4.7. I know it didn't make sense now. I have been updating a ton of mods all morning and my brain is fuzzy. We have a very quickly growing ultimate server of 40+ at a time. I added tickthreading 1457 and after doing so I am getting spammed with the same error message as him non stop. I never had any errors till adding it, and if i remove it they go away.
http://pastebin.com/xAhYa11c
